0_RootFS/Rootfs/bundled/conf/profile.d/bb_utils.sh

```diff
@@ -62,15 +62,40 @@ qfind() {
     find "$@" 2>/dev/null
 }
 
-# Function to install license files
-install_license () {
-    if [ $# -eq 0 ]; then
-        echo "Usage: install_license license_file1.txt [license_file2.md, license_file3.rtf, ...]" >&2
-        exit 1
+# Function to automatically install license files at the end of the build
+auto_install_license () {
+    if [[ -z "$(qfind "${prefix}/share/licenses/${SRC_NAME}" -mindepth 1)" ]]; then
+        # There are no licenses already installed, let's find 'em all
+
+        DIR="${WORKSPACE}/srcdir"
+        # Build the list of known names for license files
+        LICENSE_FILENAMES=()
+        for bname in COPYING COPYRIGHT LICENCE LICENSE ; do
+            for extension in "" .md .txt; do
+                # These are actually going to be options for `find`
+                LICENSE_FILENAMES+=(-iname "${bname}${extension}" -o)
+            done
+        done
+        # Remove the last OR
+        unset 'LICENSE_FILENAMES[${#LICENSE_FILENAMES[@]}-1]'
+
+        # First round: look for license files in ${DIR}
+        qfind "${DIR}" -maxdepth 1 -type f \( "${LICENSE_FILENAMES[@]}" \) \
+              -exec install_license '{}' \;
+
+        if [[ -z "$(qfind "${prefix}/share/licenses/${SRC_NAME}" -mindepth 1)" ]]; then
+            # We didn't install anything.  Let's see if there is a single subdir
+            # in ${DIR}, if so look for a license file inside it
+            dirs=$(qfind "${DIR}" -mindepth 1 -maxdepth 1 -type d -path "${DIR}/patches" -prune -o -print)
+            # This test is not safe if there are new lines in ${dirs}, but dealing
+            # with new lines here would require a bit of extra work, probably not
+            # worth the effort, as we can manually install the license if necessary
+            if [[ "$(echo "$dirs" | wc -l)" -eq 1 ]]; then
```

Comment:
Agreed that this isn't worth the effort, but if you really did want to do this, you could do [something like this](https://stackoverflow.com/a/11131443/230778)